packages/ui/Scrollable/ScrollableNative.js

```diff
@@ -728,7 +732,12 @@ class ScrollableBaseNative extends Component {
 	onScroll = (ev) => {
 		const {scrollLeft, scrollTop} = ev.target;
 
-		this.startScrollJob.startRaf(scrollLeft, scrollTop);
+		if (!this.scrollRaFId) {
+			this.scrollRaFId = window.requestAnimationFrame(() => {
```

Comment:
Is there any possibility that `scroll` event occurs twice or more times between rAF callback? If so, we may have still the same problem of ignoring the later scroll positions.